ui/src/app/Login/LoginForm.tsx

```diff
@@ -15,7 +15,13 @@ export function FormLogin() {
   async function loginHandler(uid: string, pwd: string) {
     try {
       const response = await login({ uid: uid, pwd: pwd });
-      if ('error' in response && response.error) {
+      if ('unavailable' === response.status) {
+        setLoginMessage('Your deployment was not successful, please return to Microsoft Azure Portal and redeploy.')
```

Comment:
I think we should make this message about login, not deployment.
Something like "You can not log-in because the deployment has failed. Please return..."

Reply:
Argue w/ Priya! :)  This is verbatim what she requested in the channel.

Reply:
Here is the new message provided by Priya:
`Sorry, we are having trouble signing you in as your deployment may have failed. Please return to Microsoft Azure Portal and attempt to redeploy Ansible Automation Platform".`

Reply:
OK, updating!